src/languages/es.js

```diff
@@ -351,7 +351,7 @@ export default {
         aboutPage: {
             description: 'La nueva Expensify está creada por una comunidad de desarrolladores de código abierto de todo el mundo. Ayúdanos a construir el futuro de Expensify.',
             appDownloadLinks: 'Enlaces para descargar la App',
-            viewKeyboardShortcuts: 'Ver atajos de teclado',
+            viewKeyboardShortcuts: 'Ver Atajos de Teclado',
```

Comment:
This text is uppercase in the English version 🤷‍♀️

Reply:
FWIW I think I recall seeing some copy guidelines somewhere that says _not_ To Use Title Case for UI Strings

Reply:
I can change it back, maybe the english version is the one that was wrong